aws-lc-rs/tests/data/ecdsa_verify_asn1_sha3_tests.txt

```diff
@@ -60,3 +60,113 @@ Msg = b3c63e5f5a21c4bfe3dbc644354d9a949186d6a9e1dd873828782aa6a0f1df2f64114a430b
 Q = 04004d5c8afee038984d2ea96681ec0dccb6b52dfa4ee2e2a77a23c8cf43ef19905a34d6f5d8c5cf0981ed804d89d175b17d1a63522ceb1e785c0f5a1d2f3d15e513520014368b8e746807b2b68f3615cd78d761a464ddd7918fc8df51d225962fdf1e3dc243e265100ff0ec133359e332e44dd49afd8e5f38fe86133573432d33c02fa0a3
 Sig = 308188024200ed6d28d082004a5d4259ff72baebd4b555ff9129ee66e8f82a062c50e4b9b0e006b4bd4173b32c724dcd507c012b21da32155c109b5e24d15c117cf1303897809c0242015eae6269c250de32e85fd2644f1b6db29a318e21498216bb779fc022c63165ed75ec19bf8d5af2aa3885b790423f08a9271b5614b1539d29f4ca7139ff3863222a
 Result = F (1 - Message changed)
+
+# [secp256k1,SHA3-256]
+
+# Test vectors for secp256k1 were generated by a modified version of the following go script:
+# https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/ecdsa/make_ecdsa_secp256k1_tests.go
+
+Curve = secp256k1
+Digest = SHA3-256
+Msg = 30feab0b62ddaf9c629913e1d7bc9912dfe80b159bb89255f84621312aecf160ae71c7592bd424c92bad002d6f8a347aafeb9864d981bace9687b399beda15f1811b00acb3529238129fbd49183b97ca94cb0ed24ae370b1c80704880c31fba10eec09bb1b44ab883988ee04a8de9ea258927d897b7157c3ee80d2826af68caf
+Q = 045dac5848d2ac731af182379a12ba89e52e51a21db5cdf8f0bdd654ee0dc78262e60c946f39a8a5256fbcda25b3b3be7f594390af3016c3cf74477041db7839be
+Sig = 30450220008954555654473abe43d9e1c42cdacec5a3838d2fc3282438f996bedc8f5e120221009fe375634e8fa7686e759395c213f5f45addc27512c5e854a6ee3ebb38957bea
+Result = F(4 )
```

Comment:
Do the failure number codes above mean the same thing here?

Reply:
Yeah they correspond to the same thing